app/code/Magento/Sales/Console/Command/PaymentUpdateEncryptCommand.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Sales\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+use Magento\Framework\Console\Cli;
+use Magento\Framework\Encryption\Encryptor;
+
+/**
+ * Command for updated encrypted credit card data to the latest cipher
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ */
+class PaymentUpdateEncryptCommand extends Command
+{
+    /** Command name */
+    const NAME = 'encryption:update:payment';
```

Comment:
Please follow https://devdocs.magento.com/guides/v2.2/extension-dev-guide/cli-cmds/cli-naming-guidelines.html
Maybe something like `encryption:payment-data:update`
cc @piotrekkaminski

Reply:
encryption:payment-data:update (or reencrypt) looks good to me. Is it the only CLI command? how are stored credentials for eg PayPal updated? What if we import file containing old password hashes?

Reply:
config values using `\Magento\Config\Model\Config\Backend\Encrypted` as the backend model and re-encrypted with `Magento\EncryptionKey\Setup\Patch\Data\SodiumChachaPatch`. The payment data was removed from the patch over concerns about the volume of data. Hashing functions are outside the scope of these changes and have not been altered.